mmcv/runner/hooks/lr_updater.py

```diff
@@ -282,7 +294,7 @@ def get_lr(self, runner, base_lr):
         if self.min_lr_ratio is not None:
             target_lr = base_lr * self.min_lr_ratio
         else:
-            target_lr = self.min_lr
+            target_lr = self.min_lr  # type:ignore
```

Comment:
It could be better to pre-define the type of target_lr: `target_lr: Union[float, xxx]`